.github/workflows/scheduled.yml

```
@@ -0,0 +1,60 @@
+name: Scheduled jobs
+
+on:
+  schedule:
+   - cron: "15 5 * * 1"
+
+jobs:
+  check-broken-links:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.DEPLOYMENT_PERSONAL_ACCESS_TOKEN }}
+          submodules: true
+
+      - name: Check for broken links
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.5.4
+        with:
+          args: --verbose ./docs/**/*.md ./submodules/chart/docs/**/*.md
```

I believe we can remove this

```suggestion
          args: --verbose ./docs/**/*.md
```

---

Absolutely! Thanks for catching it. 🙏